docs/_blog/2021-5-5-kuma-1-1-5.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Kuma 1.1.5 Released
+description: Kuma 1.1.5 Released
+date: 2021-5-5
+tags:
+  - Release
+canonicalUrl: 'https://konghq.com/blog/kuma-1-5-5-released'
```

Comment:
shouldn't be `1-1-5` instead of `1-5-5`?
```suggestion
canonicalUrl: 'https://konghq.com/blog/kuma-1-1-5-released'
```

Reply:
Yes @Bradamant3

Reply:
Fixed in #426 -- closing this one unmerged as not needed :D